src/org/ods/usecase/LeVADocumentUseCase.groovy

```diff
@@ -248,17 +248,23 @@ class LeVADocumentUseCase extends DocGenUseCase {
         def data_ = [
             metadata: this.getDocumentMetadata(this.DOCUMENT_TYPE_NAMES[documentType]),
             data: [
-                // TODO: change template from this.project.repositories to repositories
-                repositories: this.project.repositories,
+                repositories: this.project.repositories.collect { repo_ ->
+                    [
+                        id: repo_.id,
+                        description: repo_.metadata.description,
+                        url: repo_.url
+                    ]
+                },
                 sections: sections,
                 tests: this.project.getAutomatedTestsTypeUnit().collectEntries { testIssue ->
                     [
                         testIssue.key,
                         [
                             key: testIssue.key,
                             description: testIssue.description ?: "",
-                            // TODO: change template from isRelatedTo to systemRequirement
-                            systemRequirement: testIssue.requirements.collect{ it.key }.join(", ")
+                            systemRequirement: testIssue.requirements ? testIssue.requirements.join(", ") : "N/A",
+                            // TODO: should we only link in tech specs where the software design is defined?
```

Comment:
@clemensutschig here, it is unclear, if we can point to any linked tech spec, or if we should only consider those that have a non-empty value set in Jira?

Reply:
@metmajer  - non empty set for requirements? ... that should not happen in the first place.

Reply:
Resolved, I meant to refer to `softwareDesignSpec` instead.